Sources/Extensions/Watch/Home/MagicItemRow/WatchMagicViewRowViewModel.swift

```diff
@@ -24,8 +24,7 @@ final class WatchMagicViewRowViewModel: ObservableObject {
     }
 
     func executeItem() {
-        guard state != .loading else { return }
```

Comment:
This also enables tapping more than once on an item before waiting it to end (request from community)